### PR TITLE
DXP-55 Imporve dockerfile, update to bookworm

### PR DESCRIPTION
**Pull request by yarlson**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -39,14 +44,19 @@ FROM dependencies as build
...
+# ---- App ----
+FROM base as app
 
-COPY --from=build /usr/bin/dumb-init /usr/bin/dumb-init
 COPY --chown=node:node --from=build /home/node/app/dist .
 COPY --chown=node:node --from=build /home/node/app/prisma prisma
 COPY --chown=node:node --from=build /home/node/app/prod_node_modules node_modules
 
+# ---- Rrelase ----
+FROM base as release
```
Comment (kibertoad) [inline Dockerfile:55]: don't we need libssl install here too?

Comment (yarlson) [reply Dockerfile:55]: It's installed in the base layer, and all other layers derive from it.

**Review by kibertoad — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # ---- Base Node ----
-FROM node:20.4.0-bullseye-slim as base
+FROM node:20.4.0-bookworm-slim as base
```
Comment (kibertoad) [inline Dockerfile:2]: probably can be bumped to 20.5.1?

Comment (yarlson) [reply Dockerfile]: done

**Pull request merged**